src/main/java/com/example/crud/controller/ShopController.java

```diff
@@ -0,0 +1,51 @@
+package com.example.crud.controller;
+
+import com.example.crud.dto.param.ShopRegisterParam;
+import com.example.crud.dto.param.ShopUpdateParam;
+import com.example.crud.dto.response.ApiResponse;
+import com.example.crud.dto.shop.ShopRegisterRequest;
+import com.example.crud.dto.shop.ShopUpdateRequest;
+import com.example.crud.enums.ReturnCode;
+import com.example.crud.service.ShopService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api")
+public class ShopController {
+
+    final ShopService shopService;
+
+
+    @PostMapping("/shops/register")
```

Comment:
```suggestion
    @PostMapping("/shops")
```

rest api는 동작은 url에 표시하지 않는다.
http method로 표시한다. 
POST로 요청을 하는 것 자체가 shop 생성을 의미하기 때문에 register는 지워주셔도 됩니다.

Reply:
cd8fa58d3b914fb52ef0d69f367cf83c163e1537

해당작업에 반영